tests/comments/api.test.ts

```diff
@@ -490,7 +490,7 @@ describe('Add Comment', () => {
           body: 'This is a new comment',
           parent: 'parent-id',
           type: 'comment',
-          mentions: ['mention-user-1', 'mention-user-2'],
+          mentions: 'mentionuser1;mentionuser2',
```

Comment:
The tests passed because the code looked like it took mentions as an array, but it is actually a string. I think to improve this we can create types for the request body and add validation middlewares